internal/ordering_test.go

```diff
@@ -293,10 +293,16 @@ metadata:
 policyDefaults:
   orderPolicies: true
   namespace: my-policies
-  consolidateManifests: false
+  consolidateManifests: true
 policies:
 - name: one
   manifests:
+  - path: {{printf "%v/%v" .Dir "configmap.yaml"}}
+    name: tiger
+  - path: {{printf "%v/%v" .Dir "configmap.yaml"}}
```

Comment:
Although this `manifest path name` has "rabbit", this name is ignored because it is consolidateManifests. The configuration name will be "tiger"

Reply:
I understand I wanted to cover most of the edge cases. because base cases are covered in other unit tests

Reply:
I can say this is the **ULTIMATE** test case.

Reply:
Maybe it is good to split